e3db/client.py

```diff
@@ -1510,8 +1510,47 @@ def __write_note(self, data, options: NoteOptions, recipient_signing_key, signin
         # Assign signature from response to Note here
         return response.status_code
 
+    def read_note(self, note_id, auth_params={}, auth_headers={}) -> Note:
+        """
+        Pubic method to make a read a note by note_id.
+
+        Parameters
+        ----------
+        note_id : str
+            UUID assigned by Tozstore, used to identify a note.
+
+        auth_params : dict
+            Extra request parameters for EACP authorizations. Optional.
+
+        auth_headers : dict
+            Extra request headers for EACP authorizations. Optional
+
+        Returns
+        -------
+        e3db.Note
+            Decrypted E3DB note
+
+        """
+        if self.public_signing_key == "":
```

Comment:
prefer having this check live in the tsv1 request authenticator so as we add more methods for notes and identity (which all use tsv1) we don't have to duplicate this conditional

Reply:
Moved it into `E3DBTSV1Auth.__call__()`, it does make a lot more sense to have it there